src/main/kotlin/no/nav/familie/tilbake/api/dto/ManuellBrevmottakerRequestDto.kt

```diff
@@ -0,0 +1,44 @@
+package no.nav.familie.tilbake.api.dto
+
+import no.nav.familie.kontrakter.felles.tilbakekreving.ManuellAdresseInfo
+import no.nav.familie.kontrakter.felles.tilbakekreving.Vergetype
+import no.nav.familie.tilbake.dokumentbestilling.manuell.brevmottaker.domene.MottakerType
+import java.util.UUID
+import javax.validation.constraints.Pattern
+
+data class ManuellBrevmottakerRequestDto(
+    val type: MottakerType,
+    val navn: String,
+    val manuellAdresseInfo: ManuellAdresseInfo? = null,
+    @field:Pattern(regexp = "(^$|.{9})", message = "Organisasjonsnummer er ikke riktig")
+    val organisasjonsnummer: String? = null,
+    @field:Pattern(regexp = "(^$|.{11})", message = "PersonIdent er ikke riktig")
+    val personIdent: String? = null,
+    val vergetype: Vergetype? = null
+) {
+    init {
+        check(!(manuellAdresseInfo == null && personIdent.isNullOrBlank() && organisasjonsnummer.isNullOrBlank())) {
+            "Må ha enten manuellAdresse, personIdent eller et organisasjonsnummer."
+        }
+        if (type == MottakerType.VERGE || type == MottakerType.FULLMEKTIG) requireNotNull(vergetype)
+    }
+}
```

Comment:
Mulig denne bare kunne vært en typealias av Brevmottaker i familie-konkrakter, etter at vi tar i bruk den nye versjonen? Tror de vil være tilnærmet identiske etter det..
```suggestion
typealias ManuellBrevmottakerRequestDto = Brevmottaker
```